src/main/resources/au/com/centrumsystems/hudson/plugin/buildpipeline/extension/PipelineHeaderExtension/columnHeader.jelly

```diff
@@ -1,70 +0,0 @@
-<j:jelly xmlns:j="jelly:core"
```

Comment:
I believe your second commit wiped this entire file, along with main.css and the changes you've made to both of them.